politeiawww/cmd/piwww/editproposal.go

```diff
@@ -107,22 +121,55 @@ func (cmd *EditProposalCmd) Execute(args []string) error {
 
 		files = append(files, f)
 	}
-
 	// Setup metadata
-	if cmd.Name == "" {
-		r, err := util.Random(v1.PolicyMinProposalNameLength)
+	pm := v1.ProposalMetadata{}
```

Comment:
When you initialize a variable to its zero value, use the format `var pm v1.ProposalMetadata`.  If you're initializing a variable to an actual value, use the short declaration syntax that you have here.